### PR TITLE
Homework for using stream

### PR DESCRIPTION
**Pull request by IvanBudynkevych**
Homework my

### CODE REVIEW

File: src/main/java/co/inventorsoft/StreamHomework.java [hunk 1]
```diff
@@ -6,62 +6,37 @@
...
-     * Used to filter children and adults.
-     * Returns collection of teenagers (13-19 years old).
-     *
-     * @param people collection of people to extract teenagers
-     * @return collection of teenagers
-     */
     public List<Person> extractTeenagers(final List<Person> people) {
-        return null;
+        return people.stream()
+                .filter(Person -> Person.getAge() >= 13 && Person.getAge() <= 19)
```
Comment (stefankiv) [inline src/main/java/co/inventorsoft/StreamHomework.java]: According to java code conventions we don't use Capitalized words for naming a variable and lambda parameter in this case is exactly a variable. Please, rename it to `person` so it doesn't look like a class name.

**Review by stefankiv — CHANGES_REQUESTED** [review]
While it works fine, let's make it a bit cleaner.
Don't be afraid of imports, instead of full class names (e.g. `java.util.Objects`) import it.

I also wonder why you removed all the comments? :D

File: src/main/java/co/inventorsoft/StreamHomework.java (hunk 1 above)
Comment (IvanBudynkevych) [reply src/main/java/co/inventorsoft/StreamHomework.java]: rename the parametr

File: src/main/java/co/inventorsoft/StreamHomework.java [hunk 2]
```diff
@@ -6,62 +6,37 @@
...
-     * Used to filter children and adults.
-     * Returns collection of teenagers (13-19 years old).
-     *
-     * @param people collection of people to extract teenagers
-     * @return collection of teenagers
-     */
     public List<Person> extractTeenagers(final List<Person> people) {
-        return null;
+        return people.stream()
+                .filter(RandomeNameOfParametr -> RandomeNameOfParametr.getAge() >= 13 && RandomeNameOfParametr.getAge() <= 19)
```
Comment (stefankiv) [inline src/main/java/co/inventorsoft/StreamHomework.java:13]: Are you kidding?

File: src/main/java/co/inventorsoft/StreamHomework.java [hunk 3]
```diff
@@ -6,62 +6,37 @@
...
- */
+
 public class StreamHomework {
-    /**
-     * Used to filter children and adults.
-     * Returns collection of teenagers (13-19 years old).
-     *
-     * @param people collection of people to extract teenagers
-     * @return collection of teenagers
-     */
```
Comment (stefankiv) [inline src/main/java/co/inventorsoft/StreamHomework.java:19]: Why did you remove all the comments?

File: src/main/java/co/inventorsoft/StreamHomework.java [hunk 4]
```diff
@@ -6,62 +6,37 @@
...
     public List<Person> extractTeenagers(final List<Person> people) {
-        return null;
+        return people.stream()
+                .filter(RandomeNameOfParametr -> RandomeNameOfParametr.getAge() >= 13 && RandomeNameOfParametr.getAge() <= 19)
+                .collect(java.util.stream.Collectors.toList());
```
Comment (stefankiv) [inline src/main/java/co/inventorsoft/StreamHomework.java:14]: Please, use imports instead of full package names wherever possible.

**Review by stefankiv — CHANGES_REQUESTED** [review]
